ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/daml_kvutils.proto

```diff
@@ -183,20 +189,59 @@ message DamlTransactionBlindingInfo {
 // A transaction rejection entry.
 message DamlTransactionRejectionEntry {
   DamlSubmitterInfo submitter_info = 1;
-
   reserved 5; // was reason.maximum_record_time_exceeded
 
   oneof reason {
+    InvalidLedgerTime invalid_ledger_time = 9;
+    SubmitterCannotActViaParticipant submitter_cannot_act_via_participant = 8;
```

Comment:
Why are these two not deprecated? Are they used by both?

If they are, can you add a comment explaining this?

Reply:
Exactly, these two errors kept their usage. Will add a comment.